doc/man7/EVP_KDF_PBKDF2.pod

```diff
@@ -34,9 +34,22 @@ The supported controls are:
 
 These controls work as described in L<EVP_KDF_CTX(3)/CONTROLS>.
 
-B<iter> is the iteration count and its value should be greater than or equal to
-1. RFC 2898 suggests an iteration count of at least 1000.  The default value is
-2048.  Any B<iter> less than 1 is treated as a single iteration.
```

Comment:
We seem to have lost the information that the default number of iterations is 2048.